rtengine/camconst.json

```diff
@@ -1149,6 +1149,22 @@ Camera constants:
         "raw_crop": [ 144, 46, 6744, 4500 ]
     },
 
+    { // Quality C
+        "make_model": "Canon EOS R5",
+        "dcraw_matrix" : [9766, -2953, -1254, -4276, 12116, 2433, -437, 1336, 5131],
+        "raw_crop" : [ 124, 92, 8220, 5486 ],
+        "masked_areas" : [ 94, 20, 5578, 122 ],
+        "ranges" : { "white" : 16383 } 
```

Comment:
Based on my analysis [here](https://github.com/Beep6581/RawTherapee/issues/5901), I would recommend 16382 as a more appropriate white level.

Reply:
also changed for the R6